src/vis/map-cursor-manager.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * Changes the mouse pointer based on feature events and status
+ * of map's interactivity.
+ * @param {Object} deps Dependencies
+ */
+var MapCursorManager = function (deps) {
+  if (!deps.mapView) throw new Error('mapView is required');
+  if (!deps.mapModel) throw new Error('mapModel is required');
+  if (!deps.featureEvents) throw new Error('featureEvents is required');
+
+  this._mapView = deps.mapView;
+  this._mapModel = deps.mapModel;
+  this._featureEvents = deps.featureEvents;
+
+  this._featureEvents.on('featureOver', this._onFeatureOver, this);
+  this._featureEvents.on('featureOut', this._onFeatureOut, this);
```

Comment:
This class is not going to "die" during the session... where would I unbind these events?